app/routes/newsletters/search.js

```diff
@@ -126,8 +143,14 @@ export default class NewsletterInfosSearchRoute extends Route {
     });
   }
 
-  setupController(controller) {
-    super.setupController(...arguments);
+  setupController(controller, model) {
+    super.setupController(controller, model);
+    const params = this.paramsFor('search');
```

Comment:
Should be reworked since all params are defined on this route now and no longer on a parent route.

Reply:
true, I'll start with removing the `const searchParams = this.paramsFor('search');` since that wont do anything I imagine.